src/workloads/execution/LookupSBEPushdown.yml

```diff
@@ -436,6 +436,9 @@ AutoRun:
       $eq:
       - standalone
       - standalone-all-feature-flags
+      - standalone-classic-query-engine
```

Comment:
These aren't the only `AutoRun` configurations that would have to change, are they? I'm curious as to how these two files were chosen, since grepping for `AutoRun` in the genny repository I see 142 hits. I can't tell at a glance which other genny workloads might be relevant to the tasks we intend to run on the classic engine build variant...

Reply:
These two files contains "SBE" in file names, so I guess these are the only queries that could be run on SBE engine. 
There are only four tasks run on "standalone-all-feature-flags", the other two are:

- ColumnStoreIndex.yml
- TimeSeriesLastpoint.yml

Reply:
> These two files contains "SBE" in file names, so I guess these are the only queries that could be run on SBE engine.

I'm not sure that is true. When `featureFlagSbeFull` is on, as it is in the "all feature flags" build variants, then most queries should be using SBE. I think we want to make sure that pretty much all sys-perf benchmarks that are relevant to query processing run in the new classic engine build variant. Therefore, I suspect that there are additional `AutoRun` configurations in the genny repository that need to change.

@dalyd do you have any tips for how to track down the right set of files? Should we just independently triage all Genny workloads and try to figure out which ones should be running on the classic engine build variant?

Another question: I think some Genny workloads run in sys-perf and some in performance, is that right? Assuming that is correct, can this patch preemptively modify all of the `AutoRun` sections for benchmarks related to query processing, even for those workloads that run in the performance project rather than sys-perf?

Reply:
@allenwux has a nice if experimental tool to view documentation for all the Genny workloads. That might be helpful here. 

Yes, some genny tests run on performance. It makes sense to me to update those also. I'm not entirely sure how to find them. At least some of them are explicitly defined in perf.yml. https://github.com/mongodb/mongo/blob/fecef7a1f75e196a24715fabb0721124e71e170b/etc/perf.yml#L475 
I do see a task generation code in perf.yml but I don't see a matching task, so maybe it's not used. That might be worth checking in #performance-tooling-users.